### PR TITLE
SW-2514 Update links in accession edit

### PR DESCRIPTION
**Pull request by karthikbtf**
- also use + icon
- introduced an AddLink component

Terraware App 2022-12-09 08-33-07

Terraware App 2022-12-09 10-07-06

Terraware App 2022-12-09 08-22-38

Terraware App 2022-12-09 08-22-08

### CODE REVIEW

File: src/components/common/AddLink.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+type AddLinkProps = {
+  text: string;
+  id?: string;
+  large?: boolean;
+  onClick: (args: any) => void;
+  disabled?: boolean;
+};
+
+export default function AddLink(props: AddLinkProps): JSX.Element {
```
Comment (charlie-tf) [inline src/components/common/AddLink.tsx:27]: nice component!

File: src/components/common/AddLink.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import { Box, Theme } from '@mui/material';
+import { Icon } from '@terraware/web-components';
+import Link from 'src/components/common/Link';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  addIcon: {
+    fill: theme.palette.TwClrIcnBrand,
+  },
+  disabledIcon: {
+    fill: theme.palette.TwClrBaseGray300,
```
Comment (charlie-tf) [inline src/components/common/AddLink.tsx:11]: base color instead of a semantic one?

Comment (hannah-cai) [reply src/components/common/AddLink.tsx:11]: we're not really using disabled icons, what is this for?

**Review by hannah-cai — APPROVED** [review]

File: src/components/common/AddLink.tsx (hunk 2 above)
Comment (karthikbtf) [reply src/components/common/AddLink.tsx:11]: @hannah-cai here's the use-case (a completed viability test)
Terraware App 2022-12-09 13-26-18

Comment (karthikbtf) [reply src/components/common/AddLink.tsx:11]: @charlie-tf good catch, i copied it without checking for correctness. will update both

Comment (karthikbtf) [reply src/components/common/AddLink.tsx:11]: removing the color change, just using opacity 0.5 as per our design

File: src/components/common/AddLink.tsx (hunk 1 above)
Comment (karthikbtf) [reply src/components/common/AddLink.tsx:27]: Thanks @charlie-tf - I also added a few others like Link.tsx, TitleDescription.tsx, Card.tsx .. which seem reusable with specs. I can move the useful ones from that list to web-components at some point.

File: src/components/common/AddLink.tsx (hunk 2 above)
Comment (karthikbtf) [reply src/components/common/AddLink.tsx]: Terraware App 2022-12-09 13-31-18

**Pull request merged**